src/components/Footer.astro

```diff
@@ -8,41 +10,205 @@
           target="_blank"
         >
           <span
-            class="self-center text-2xl font-semibold whitespace-nowrap text-white"
+            class="font-header self-center text-2xl font-semibold whitespace-nowrap text-white"
             >LFX Mentorship</span
           >
         </a>
       </div>
     </div>
-    <hr class="my-6 sm:mx-auto border-gray-500 lg:my-8" />
+
+    <hr class="my-6 sm:mx-auto border-highlight-blue lg:my-8 border-t-2" />
+
     <div
-      class="flex gap-2 items-end justify-between md:items-center md:justify-between"
+      class="flex gap-2 items-center justify-between flex-col sm:items-end sm:flex-row md:items-center md:justify-between"
     >
-      <span class="text-sm text-gray-500 text-center dark:text-gray-400">
-        <a
-          href="https://www.copsiitbhu.co.in/"
-          class="hover:underline"
-          target="_blank">COPS @IIT(BHU)-Varanasi</a
-        >
+      <span
+        class="font-subheader text-xl text-gray-500 text-center dark:text-gray-400"
+      >
+        <a href="https://www.copsiitbhu.co.in/" class="glitch" target="_blank">
+          <span class="mainText skew hover:underline"
+            >COPS @ IIT(BHU)
+            <span class="backText glitch hover:underline">COPS @ IIT(BHU)</span>
+          </span>
+        </a>
       </span>
+
+      <span
+        class="font-subheader text-xl text-gray-500 text-center dark:text-gray-400"
+      >
+        <a href="https://sdg.copsiitbhu.co.in/" class="glitch" target="_blank">
+          <span class="mainText skew hover:underline"
+            >COPS SDG
```

Comment:
Replace with somewhat of a fancier font and link to the [COPS SDG website](https://sdg.copsiitbhu.co.in/)